taichi/transforms/make_adjoint.cpp

```diff
@@ -201,6 +210,11 @@ class MakeAdjoint : public IRVisitor {
       auto numerator = add(sqr(bin->lhs), sqr(bin->rhs));
       accumulate(bin->lhs, div(mul(adjoint(bin), bin->rhs), numerator));
       accumulate(bin->rhs, negate(div(mul(adjoint(bin), bin->lhs), numerator)));
+    } else if (bin->op_type == BinaryOpType::pow) {
+      // d (x ^ y) = x ^ (y-1) * (y * dx + log(x) * x * dy)
+      auto numerator = pow(bin->lhs, sub(bin->rhs, constant(1)));
```

Comment:
`numerator` has a special meaning. How about `common_coeff`? It would also be helpful to append `// x ^ (y-1)` to this line.